src/net/sourceforge/kolmafia/request/RelayRequest.java

```diff
@@ -3194,7 +3198,7 @@ private void submitCommand(String command, boolean suppressUpdate, boolean waitF
 
   public void waitForCommandCompletion() {
     while (CommandDisplayFrame.hasQueuedCommands()) {
-      this.pauser.pause(500);
+      this.pauser.pause(50);
```

Comment:
hopefully this will improve relay perf elsewhere too :)

Reply:
One hopes.

It means that any fast-executed side-command is detected more quickly.

The "step" test was painfully slow when simulating 67 side commands, with all the I/O going through our "fake" client. Essentially all the execution time was spent pausing. :)